<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-5ICULF9D" ignored="0" name="Test" tests="7" failures="2" timestamp="2021-04-19T17:46:41 IST" time="81.922" errors="0">
  <testcase name="loginTest" time="4.439" classname="com.inetBanking.testCases.TC_LoginTest_001"/>
  <testcase name="loginDDT" time="7.501" classname="com.inetBanking.testCases.TC_LoginDDT_002"/>
  <testcase name="loginDDT" time="2.972" classname="com.inetBanking.testCases.TC_LoginDDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetBanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:29)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="loginDDT" time="6.339" classname="com.inetBanking.testCases.TC_LoginDDT_002"/>
  <testcase name="loginDDT" time="4.629" classname="com.inetBanking.testCases.TC_LoginDDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetBanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:29)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="loginDDT" time="13.085" classname="com.inetBanking.testCases.TC_LoginDDT_002"/>
  <testcase name="addNewCustomer" time="16.322" classname="com.inetBanking.testCases.TC_AddCustomerTest_003"/>
</testsuite> <!-- Test -->
